styles.css

```diff
@@ -0,0 +1,246 @@
+:root{
+    --bravo: #EAB308;
+    --bravo-circle-shadow: rgba(255, 178, 54, 0.65);
+    --bravo-shadow: rgba(234, 179, 8, 0.3);
+
+    --alpha: #EA580C;
+    --alpha-circle-shadow: rgba(249, 99, 50, 0.65);
+    --alpha-shadow: rgba(234, 88, 12, 0.3);
+
+    --charlie: #22C55E;
+    --charlie-circle-shadow: rgba(22, 199, 13, 0.65);
+    --charlie-shadow: rgba(34, 197, 94, 0.3);
+
+    --extra: #EF4444;
+    --extra-circle-shadow: rgba(255, 54, 54, 0.65);
+    --extra-shadow: rgba(239, 68, 68, 0.3);
+
+    --featured: #90949F;
+    --featured-price: #1C202E;
+    --featured-shadow: rgba(255, 255, 255, 0.1);
+
+    --white: #FCFCFC;
+}
+
+body {
+    font-family: sans-serif;
+    /* background-color: brown; */
+    margin: 0;
+    padding: 0;
+    background-image: 
+    linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), 
+    url('./media/background.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+    background-position: center;
+  }
+
+.main-container {
+    display: flex;
+    flex-direction: column;
+    gap: 60px;
+    /* background-color: blue; */
+    /* padding: 25px; */
+    padding: 70px 0 70px 0;
+    width: 80%;
+    text-align: center;
+    color: var(--white);
+    font-weight: 400;
+    margin: auto;
+
+}
+
+.titles{
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    padding: 0;
+    gap: 30px;
+}
+
+h1 {
+    font-size: 36px;
+    margin: 0;
+    padding: 0;
+    font-family: 'Ubuntu', sans-serif;
+    font-weight: 500;
+    /* margin-top: 20px; */
+    /* background-color: blueviolet; */
+  }
+
+h3 {
+    font-size: 22px;
+    margin: auto;
+    padding: 0;
+    font-family: 'Manrope', sans-serif;
+    font-weight: 400;
+    width: 75%;
+    max-width: 540px;
+    text-align: center;
+    /* background-color: cornsilk; */
+  }
+
+.pricing-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, 255px);
+    grid-gap: 30px;
+    justify-content: center;
+    font-family: 'Manrope', sans-serif;
+    /* padding-top: 30px;
+    padding-bottom: 75px; */
+}
+
+h2 {
+    font-weight: 400;
+    font-size: 18px;
+}
+
+hr {
+    border-top: 1px solid #D1D2D9;
+    width: 100%;
+    gap: 0px;
+    padding: 0;
+    margin: 0;
+}
+
+.pricing-grid p {
+    size: 14px;    
+}
+
+.price-tag {
+    font-size: 22px;
+}
+
+.pricing-grid button {
+    font-family: 'Manrope', sans-serif;
+    width: 115px;
+    height: 38px;
+    border-radius: 50px;
+    margin-bottom: 15px;
+    margin: 15px 0 15px 0;
+    border: 0;
+}
+
+.pricing-grid button:hover {
+    transform: scale(1.05);
+    cursor: pointer;
+}
+  
+.pack {
+    /* background-color: chartreuse; */
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 0px;
```

Comment:
I see that you are not using gap to separate items inside pack, and you are using the paddings of the elements.
Plase try to delete all the padding, for example of the `<p>` and the `<h>` and use the gap!

Reply:
@thomasmcculloch2 here was I was trying to achieve was to add space between the border of the element (pack) and not trying to add further space between each element inside pack. 
<img width="821" alt="Monosnap Untitled presentation - Google Slides 2023-11-27 12-35-10" src="https://github.com/pochi-lightit/fe-challenge-2/assets/149687080/5173ae4d-aa32-4ff6-b9c7-9eb98fafb472">

It's similar to the situation above, how do I add space on top or below the first or last element using gap?

Reply:
Lets review this in the meet tomorrow!